src/hacking/building-for-openharmony.md

```diff
@@ -4,189 +4,29 @@
 </div>
 
-## Get the OpenHarmony tools
+The building for servo is currently done in two steps and needs a Linux workstation and MacOS/Windows computer for bundling.
+1. Build the shared library (libservo.so).
+2. Create the App Bundle.
 
-Building for OpenHarmony requires the following:
-
-1. The OpenHarmony SDK. This is sufficient to compile servo as a shared library for OpenHarmony.
-2. The `hvigor` build tool to compile an application into an app bundle and sign it.
-
-### Setting up the OpenHarmony SDK
-
-The OpenHarmony SDK is required to compile applications for OpenHarmony.
-The minimum version of SDK that Servo currently supports is v5.0 (API-12).
-
-#### Downloading via DevEco Studio
-
-[DevEco Studio] is an IDE for developing applications for HarmonyOS NEXT and OpenHarmony.
-It supports Windows and MacOS.
-You can manage installed OpenHarmony SDKs by clicking File->Settings and selecting "OpenHarmony SDK".
-After setting a suitable installation path, you can select the components you want to install for each available API version.
-DevEco Studio will automatically download and install the components for you.
-
-
-#### Manual installation of the OpenHarmony SDK (e.g. on Linux)
-
-<div class="warning _note">
-    Before rushing and downloading the OH SDK from gitee as described here, please note that you will also need hvigor to compile applications.
-    hvigor is currently recommended to be downloaded via the HarmonyOS NEXT commandline tools package, which also contains a copy of the OpenHarmony SDK.
-</div>
-
-1. Go to the [OpenHarmony release notes] and select the version you want to compile for.
+## Step 1
+### Prerequisites
+1. Go to the [OpenHarmony release notes](https://gitee.com/openharmony/docs/blob/master/zh-cn/release-notes/Readme.md) and select the version you want to compile for.
 2. Scroll down to the section "Acquiring Source Code from Mirrors" and click the download link for the version of "Public SDK package for the standard system" matching your host system.
 3. Extract the archive to a suitable location.
 4. Switch into the SDK folder with `cd <sdk_folder>/<your_operating_system>`.
 5. Create a sub-folder with the same name as the API version (e.g 12 for SDK v5.0) and switch into it.
 6. Unzip the zip files of the individual components into the folder created in the previous step. Preferably use the `unzip` command on the command-line, or manually ensure that the unzipped bundles are called e.g. `native` and not `native-linux-x64-5.x.y.z`.
 
-The following snippet can be used as a reference for steps 4-6:
-```commandline
-cd ~/ohos-sdk/linux
-for COMPONENT in "native toolchains ets js previewer" do
-    echo "Extracting component ${COMPONENT}"
-    unzip ${COMPONENT}-*.zip
-    API_VERSION=$(cat ${COMPONENT}/oh-uni-package.json | jq -r '.apiVersion')
-    mkdir -p ${API_VERSION}
-    mv ${COMPONENT} "${API_VERSION}/"
-done
```

Comment:
hvigor is very finicky with the folder layout. This is necessary when downloading the OH SDK (as opposed to the harmonyos commandline tools, which are fine by default)